Problem4/prob4.py

```diff
@@ -0,0 +1,25 @@
+def missing_character(alphaList):
+    if not alphaList or len(alphaList) < 2:
+        return None
+    
+    if not all(isinstance(c, str) and len(c) == 1 for c in alphaList):
+        return None
+        
+    if not all(c.isalpha() for c in alphaList):
+        return None
+        
+    first_char_is_upper = alphaList[0].isupper()
+    if not all(c.isupper() == first_char_is_upper for c in alphaList):
+        return None
+    
+    alphaList.sort()
```

Comment:
I don't think we need to sort either as the problem clearly states
`takes an array of consecutive (increasing) letters as input`